### PR TITLE
removed 'jf-is-scrollable-if-too-tall' class from privacy-modal div

### PR DESCRIPTION
**Pull request by Sjones21**
Solves #305 by removing the 'jf-is-scrollable-if-too-tall' fix (ce8b4c1) from #101. Privacy modal now has one scrollbar when screen is small on desktop and mobile, in both portrait mode and landscape.

### CODE REVIEW

**Comment by Sjones21** [comment]
@toolness This PR seems to be in a great spot! The mobile scrolling issues are certainly solved, and the modal is scrollable and exitable on every single web browser I could test on. The only issue I could not resolve is that some newer browsers will add a second vertical scrollbar when the browser window height is less than 400px or width is less than 500px. This extra outer scrollbar doesn't interfere with usability too much, you can still scroll down and/or exit the modal, and the screen is really narrow at that size.

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "babel-loader": "8.0.5",
     "babel-plugin-dynamic-import-node": "2.2.0",
     "babel-plugin-macros": "2.8.0",
-    "bulma": "0.7.4",
+    "bulma": "0.9.0>",
```
Comment (toolness) [inline package.json:119]: Is this a requirement for your fix to work?  If not, I think we should maybe leave it for a separate PR--one where we can make sure that none of the site layout changes when we upgrade!

File: frontend/sass/_modal.scss [hunk 2]
```diff
@@ -22,21 +22,52 @@
   flex-direction: column;
 }
 
+// This styling applies to desktop Internet Explorer
+.scrollable {
```
Comment (toolness) [inline frontend/sass/_modal.scss:26]: Did you add this class anew, or is it building on a pre-existing class from Bulma?  If the former, can you prefix this with `jf-`?  That stands for `justfix-`, and we're trying to use it for all our own first-party CSS classes to make it easy to identify which classes are ours, rather than Bulma's.

File: frontend/sass/_modal.scss [hunk 3]
```diff
@@ -22,21 +22,52 @@
...
+  overflow-y: auto;
+  max-width: 640px;
+}
+
 .jf-modal-dialog {
   cursor: default;
+  overflow-y: initial !important;
```
Comment (toolness) [inline frontend/sass/_modal.scss:33]: Is there any way to make this rule without the `!important`?  It's generally preferable to avoid it, though there [are some exceptions](https://css-tricks.com/when-using-important-is-the-right-choice/).

File: frontend/sass/_modal.scss [hunk 4]
```diff
@@ -22,21 +22,52 @@
...
+    height: 250px;
+    max-width: 75vw;
+  }
+  .jf-modal-dialog {
+    position: absolute;
+    max-width: 75vw;
+    left: 28%;
+    left: calc(50% - 320px);
+    top: 15%;
   }
```
Comment (toolness) [inline frontend/sass/_modal.scss:72]: It seems like there's a lot of repetition between this rule and another one about 10 lines above it.  Do you think it can be refactored to make things more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), either via standard CSS or SCSS (like a [mixin or include](https://sass-lang.com/documentation/at-rules/mixin))?

**Comment by Sjones21** [comment]
@toolness I actually went back to your original "is-scrollable-if-too-tall" class and figured out a way to hide all other scrollbars! I also added one media query to help the width of the modal on mobile. Modals are not scrollable on older versions of desktop (Safari 5.1, Chrome 33, Firefox 18), so in that case the user needs to resize the window. Also, Bulma is now listed at v.0.7.4 in the package.json file.

**Comment by toolness** [comment]
Thanks Sydney!  This looks much better.

Hmm, so the one thing I'm worried about now is that if we don't apply that "is-scrollable-if-too-tall" class, while the hiding of the scrollbars is nice, it actually potentially makes things un-scrollable in some (super edge-casey) situations.  For instance, if someone has a relatively low-res screen, is visually impaired and needs to increase the size of the text in their browser, this is currently what it the "no lease" modal looks like _without_ your fix:

> image

That is all kinds of bad, but at least there's a scrollbar on the right-hand side that allows the user to read the lines one by one, and click the button at the very bottom.

In contrast, this is what the same modal looks like with your fix:

> image

Yours is an improvement in that the modal at least fills the whole screen so the user isn't forced to read it line-by-line, but they also can't scroll it, which is a big bummer.

(By the way, the above screenshots were taken in a recent version of Firefox, though I confirmed the same behavior occurs in recent versions of Chrome too.)

So even though the removal of Bulma's default scroll bar looks nice on _most_ browsers, I'm wondering if we should actually keep it, to ensure that _all_ users will be able to scroll through the content?

I'm also wondering how hard it is to duplicate the Bootstrap modal styling completely, so that the browser window's scroll bar actually just scrolls the modal.  That definitely feels like the most aesthetically pleasing _and_ most usable solution...

Anyhow, let me know what you think--sorry it took me so long to respond to this!

**Comment by toolness** [comment]
Acck sorry didn't mean to close this, reopening.

**Comment by toolness** [comment]
Unfortunately this PR still has issues and we need to clean up our list of PRs, so I'm going to close this for now.
